src/manifold/src/smoothing.cpp

```diff
@@ -308,6 +191,296 @@ struct InterpTri {
     pos = HNormalize(posH);
   }
 };
+
+class Partition {
+ public:
+  glm::ivec3 idx;
```

Comment:
The cached partitions don't have this - it's added to the copy returned from `GetPartition` that contains the mapping of the input divisions into the sorted divisions that are uniquely cached.